frontend/locales/pl/faq.json

```diff
@@ -0,0 +1,29 @@
+{
+    "faq": {
+      "page_title": "LionSkins - FAQ",
+      "breadcrumb": "FAQ",
+      "title": "FAQ",
+      "subtitle": "Masz pytanie?",
+      "part1": {
+        "title": "Dlaczego tylko Counter-Strike?",
+        "content": "Na początku chcieliśmy dodać więcej gier, ale nie mieliśmy na to ani czasu, ani środków... Ale pamiętamy o tym, może kiedyś to nastąpi!"
+      },
+      "part2": {
+        "title": "Dlaczego nie sugerujecie danego rynku?",
+        "content": "Na pewno strona którą sugerujesz jest wspaniała, ale nie możemy uwzględnić wszystkich możliwych rynków. Jeżeli jednak będzie liczny odzew a sprawdzimy że produkt który oferujesz jest na pewno bezpieczny być może go dodamy w przyszłości."
+      },
+      "part3": {
+        "title": "Podana cena nie była właściwa!",
+        "content": "Mimo że dążymy do aktualizacji cen w czasie rzeczywistym, nie jesteśmy akutalnie w stanie tego zrobić ze względów technicznych."
```

Comment:
Part of the translation seems to have been missed

Reply:
Yeah, I shortened this sentence strongly. It had the same meaning, but I added the missing part.